README.md

```diff
@@ -1,42 +1,61 @@
-Thank you very much for taking the time to report a bug to us, we greatly
-appreciate it. Even more so, since you are about to take the time to create an
-[MCVE (Minimal Complete Verifiable Example)](https://stackoverflow.com/help/minimal-reproducible-example). Thanks to you, we can make jOOQ an
-even better product!
+Thank you very much for taking the time to report a bug to us, we greatly appreciate it.
+Even more so, since you are about to take the time to create an
+[MCVE (Minimal Complete Verifiable Example)](https://stackoverflow.com/help/minimal-reproducible-example).
+Thanks to you, we can make jOOQ an even better product!
+
+
+This project contains —for each supported language (Java, Kotlin, Scala) and SQL dialect (H2, MySQL, Oracle, PostgreSQL, SQL Server)—
+an minimal project from which you can derive your MCVE. Each of these projects sits in its own folder and:
+
+- Uses testcontainers (for MySQL, Oracle, PostgreSQL, SQL Server) to set up a database instance, and makes it available to jOOQ and JUnit.
+- Installs a sample schema located in `src/main/resources/db/migration` into an file-based H2 database or testcontainers managed MySQL, Oracle, PostgreSQL, SQL Server database.
+- Runs jOOQ's code generator on it.
+- Runs a simple integration test.
+
+This should work without any additional setup on your side, other than having a JDK, build tool (only when using Maven), and Docker available.
+If you need to have the commercial distribution of jOOQ installed, e.g. for using Oracle or SQL Server, that must be installed as well.
+In case you do not have Docker, you can still run the H2 based examples directly, ignoring the MySQL, Oracle, PostgreSQL, SQL Server based ones.
+
```

Comment:
I understand why you wrote this, and for some people who like to read Macbeth in its original text, this is nice. But for most people, I'm positive it's just noise. They know why they're doing this, so they just want to get the minimal information they need.

Reply:
I just realise, I wrote this myself, further down 😅 Jeez, I wonder if we really need it.